pyproject.toml

```
@@ -13,14 +13,20 @@ python = "^3.10"
 click = "^8.1.7"
 pyserial = "^3.5"
 pyusb = "^1.2.1"
-pyudev = "^0.24.3"
 httpx = "^0.27.0"
 pyyaml = "^6.0.1"
 anyio = "^4.4.0"
 typing-extensions = "^4.12.2"
-gpiozero = { version = "^2.0.1", optional = true }
 aiohttp = "^3.9.5"
 
+# rpi
+gpiozero = { version = "^2.0.1", optional = true }
+
+# dutlink
+libusb = {version = "^1.0.27", optional = true}
+pyudev = {version = "^0.24.3", optional = true}
+pexpect = "^4.9.0"
```

```suggestion
pexpect = "^4.9.0"
```

This one should probably go along always by default (related to pyserial usage in general)

---

`poetry add` always adds things to the botton...